themes/edsonarios/edsonarios.theme.sh

```diff
@@ -0,0 +1,19 @@
+#! bash oh-my-bash.module
+# Background: #333333
+# For unstaged(*) and staged(+) values next to branch name in __git_ps1
+GIT_PS1_SHOWDIRTYSTATE="enabled"
+OMB_PROMPT_VIRTUALENV_FORMAT=' [%s]'
+OMB_PROMPT_CONDAENV_FORMAT=' [%s]'
+
+STATUS_THEME_PROMPT_BAD="${_omb_prompt_bold_brown}❯_${_omb_prompt_normal} "
+STATUS_THEME_PROMPT_OK="${_omb_prompt_bold_green}❯_${_omb_prompt_normal} "
```

Comment:
The variable names specific to this theme should start with `OMB_THEME_` such as `OMB_THEME_EDSONARIOS_STATUS_BAD`. We have these variable names in the `purity` theme, but we want to shift to organized variable names (#280). Not to cause unnecessary compatibility issues in the future, we would like to apply the naming convention from the beginning for the new themes.

Reply:
Done